doc/reference/std/getopt.md

```diff
@@ -1,188 +1,17 @@
 # Command Line Argument Parsing
 
-The `:std/getopt` library provides facilities for command line argument parsing.
+This is the old name of the `:std/cli/getopt` module,
```

Comment:
link to the new page please, this will get all of hits.

Also add a comment saying "Up to Gerbil v0.18 this was the canonical path for the getopt library; starting with the v0.19 cycle, the module has been moved to std/cli/getopt` or some such.

Reply:
How's the new paragraph?